scene/3d/soft_body_3d.cpp

```diff
@@ -361,6 +361,12 @@ void SoftBody3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_drag_coefficient", "drag_coefficient"), &SoftBody3D::set_drag_coefficient);
 	ClassDB::bind_method(D_METHOD("get_drag_coefficient"), &SoftBody3D::get_drag_coefficient);
 
+	ClassDB::bind_method(D_METHOD("get_point_transform", "point_index"), &SoftBody3D::get_point_transform);
+
+	ClassDB::bind_method(D_METHOD("pin_point_toggle", "point_index"), &SoftBody3D::pin_point_toggle);
+	ClassDB::bind_method(D_METHOD("pin_point", "point_index", "pin", "spatial_attachment_path"), &SoftBody3D::pin_point);
```

Comment:
You can make the function names consistent for scripts and add the default value like this (so when you pinned is false you don't have to set the attachment path):
```suggestion
	ClassDB::bind_method(D_METHOD("set_point_pinned", "point_index", "pinned", "spatial_attachment_path", DEFVAL(NodePath())), &SoftBody3D::pin_point);
```

That would only rename the script function.
You can also rename the method in SoftBody3D class to clean things up, but it's not necessary for this PR.

Reply:
If I'm not mistake, this rename will also update the mono and GDNative APIs, keeping them all consistent?

Reply:
Yep, it will also update other script APIs but I don't think there's anything you need to worry about.